art/readme.txt

```diff
@@ -21,7 +21,9 @@ If you downloaded this from any other place than ninja-muffin24.itch.io/FUNKIN
 You might be in DANGER!!! 
 
 The Itch.io release is the only official source for the desktop (PC, Mac, Linux) versions of the game!
-As of right now, the game is FREE! If you paid for it, you got SCAMMED!
+As of right now, the game is FREE! If you paid for it, you got SCAMMED! The only official paid versions are on itch.io and Steam, and are of the FULL ASS GAME which does not exist as of November, 15, 2022
```

Comment:
correction: the only place that you could have paid for the full game currently is Kickstarter, which you cant even pay for it anymore because the kickstarter is over. itch.io is just donations right now, and i dont think weve heard anything about a steam release

Reply:
whoops meant it as a single comment not a reply